src/screens/FeedsManager/FeedsManagerScreen.tsx

```diff
@@ -1,24 +1,84 @@
 import React from 'react'
 
+import { gql, useMutation } from '@apollo/client'
+import { notifyErrorMsg, notifySuccessMsg } from 'actionCreators'
+import { useDispatch } from 'react-redux'
 import { Redirect, useParams } from 'react-router-dom'
 import { GraphqlErrorHandler } from 'src/components/ErrorHandler/GraphqlErrorHandler'
 import { Loading } from 'src/components/Feedback/Loading'
 import { useFeedsManagerWithProposalsQuery } from 'src/hooks/queries/useFeedsManagerWithProposalsQuery'
+import { useMutationErrorHandler } from 'src/hooks/useMutationErrorHandler'
 import NotFound from 'src/pages/NotFound'
 import { FeedsManagerView } from './FeedsManagerView'
 
+export const ENABLE_FEEDS_MANAGER_MUTATION = gql`
+  mutation EnableFeedsManager($id: ID!) {
+    enableFeedsManager(id: $id) {
+      ... on EnableFeedsManagerSuccess {
+        feedsManager {
+          id
+          name
+          uri
+          publicKey
+          isConnectionActive
+          createdAt
+          disabledAt
+        }
+      }
+      ... on NotFoundError {
+        message
+        code
+      }
+    }
+  }
+`
+
+export const DISABLE_FEEDS_MANAGER_MUTATION = gql`
+  mutation DisableFeedsManager($id: ID!) {
+    disableFeedsManager(id: $id) {
+      ... on DisableFeedsManagerSuccess {
+        feedsManager {
+          id
+          name
+          uri
+          publicKey
+          isConnectionActive
+          createdAt
+          disabledAt
```

Comment:
do we need to fetch the full information from feedsManager? eg do we need uri publickey etc?

Reply:
Yeah, this is the most up to date manager. In the rare case that someone update in the API directly the user would receive the updated manager.
I don't think first-level resolvers place a significant burden on the server either.

Reply:
ah okay make sense!